crates/bevy_window/src/window.rs

```diff
@@ -42,6 +43,7 @@ pub struct Window {
     decorations: bool,
     cursor_visible: bool,
     cursor_locked: bool,
+    pub(crate) cursor_position: Vec2,
```

Comment:
This should be `Option<Vec2>` because there are points where a window will not have a cursor position due to the cursor not being over the window.

Reply:
Additionally we may have to add a way to check if the cursor has moved off the window, unless there's already one I can't find. Something like `WindowUnfocused`.

Reply:
Good calls!

Reply:
I don't see a way of detecting the cursor has left the window.

Reply:
You'd have to add it most likely :(

Reply:
Ok. There are `CursorEntered` and `CursorLeft`.